minvws/nl-kat-coordination
# Create scheduled report with zero objects selectable

### Changes
This PR makes the following possible:
- A user can always see and select report types, even when there are no objects that meet the live filter
   - The report types that are visible are based on the ooi_types that the user selected in the filter
- A user can now create a scheduled report based on a live set without objects
- The empty reports with 0 input objects will be generated and showed on the Scheduled Reports Page, but not on the History Page (this is in line with a future design).

### Issue link
Closes #3886

### Demo

https://github.com/user-attachments/assets/ad78b7a6-d0e1-4601-87c0-4079e2f0664b

![afbeelding](https://github.com/user-attachments/assets/877f62f9-ca3c-4f6a-ab91-3384642ed37f)




### QA notes
- Create a normal Report and Aggregate Report
- Change the filter so that no objects match the filter
- You should still be able to continue
- No empty reports should be visible on the Report History page
- The empty reports should be visible (but not clickable) on the Scheduled Reports page

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
